src/hooks/useGetQuest.ts

```diff
@@ -21,6 +20,15 @@ export function useGetQuest(
     refetchOnWindowFocus: false,
     enabled: questId !== null
   })
+}
+
+export function useGetQuest(
+  questId: number | null,
+  getQuest: (questId: number) => Promise<Quest>
+) {
+  const queryClient = useQueryClient()
+  const queryKey = getGetQuestQueryKey(questId)
```

Comment:
same comment from the other hook

Reply:
Just updated to get the query key from the options.